modules/tracing/mule-tracer-api/src/main/java/org/mule/runtime/tracer/api/span/exporter/SpanExporter.java

```diff
@@ -110,7 +110,7 @@ default void onError(InternalSpanError error) {}
   /**
    * Updates parent span from a map
    *
-   * @param serializeAsMap the serialization map
+   * @param serializeAsMap the serialization map that represents the span in W3C format.
    */
   default void updateParentSpanFrom(Map<String, String> serializeAsMap) {}
```

Comment:
Thanks for adding the jdoc. I think changing the name to spanSerializedAsMap would add a bit more clarity to the variable, would it be possible to change it?

Reply:
Updated.